src/workerd/io/worker.c++

```diff
@@ -1438,6 +1440,16 @@ Worker::~Worker() noexcept(false) {
   lock->push(kj::mv(impl));
 }
 
+void Worker::logBotManagementUse() const {
+  // TODO(maybe): If logging this for every worker proves to be too noisy, then we
+  // can figure out a way to log of for workers that access when they do not have
+  // a subscription.
+  if (!impl->loggedBotManagementAccess) {
+    impl->loggedBotManagementAccess = true;
+    KJ_LOG(INFO, "Worker is accessing request.cf.botManagement metadata");
```

Comment:
INFO-level logging is not enabled in production, so this won't log anything. You'll need to upgrade it to WARNING.

Reply:
oh right, thanks for the reminder :-)